apps/checkout-app/jest.config.ts

```diff
@@ -63,9 +63,9 @@ export default {
   // globalTeardown: undefined,
 
   // A set of global variables that need to be available in all test environments
-  globals: {
-    fetch: () => undefined,
-  },
+  // globals: {
+  //   fetch: () => undefined,
+  // },
```

Comment:
Do we need to keep dead code?

Reply:
Nope, removed all the comments from Jest configuration